### ISSUE TITLE
It's not possible to call a REST Service deployed on a onpremise node diferent to kobo´s node

### ISSUE THREAD

**Issue by paolagrisales** [kobotoolbox/kpi#2788]
## Description

I have Kobo installed on a node "one", and also a wso2 micro integrator service used to put a json message in a queue. This wso2 service is over a node "two".  I´m trying to invoque this wso2 service as a REST Service, however I get this errror: after a submission the status is pending and the informative message says:
![image](https://user-images.githubusercontent.com/5765016/92631710-5cda1100-f297-11ea-9a26-3f964588ad59.png)

I have tried many things as use the node two's IP, an alias mapped in the etc/host, even config an Nginx to route my service... but nothing has worked so far. Can you tell me what features should have the Endpoint URL when I configure the REST Service in KoBoToolbox.

Note: I used ngrok also and worked, but it´s no posible to use ngrok anymore because we would need licences.

Thanks for your help.

## Steps to Reproduce

1. Create a rest service with POST; deploy it in an onpremise installation (node "two")
2. Create a sample Kobo form, using an onpremise deployment (node "one")
3. Configure the form to call the exposed service (on node "two") as a hook
4. Send the form

## Expected behavior 

Request sent sucess

## Actual behavior 

Request not sent to the hook.

We don´t know the way to configure the Endpoint URL. Thanks for your help.

## Additional details

Environment: docker deploy over ubuntu 20.04 LTS on premise installation

**Comment by jnm** [comment]
Hi, I was about to say that I'm fairly confident this is an issue with SSRF protection, but then I saw that you'd also opened #2697. Did you add your IP to `allowed_ip_addresses` in `SSRF_OPTIONS` already?

**Comment by paolagrisales** [comment]
Hi, effectively at the beginning we guessed our issue was related with to call a service deployed in the same location of kobo node and moved our component to other node, but after we understood how SSRF is working and we realized it also restricts to invocate a Private-Use IP. Thats the reason why our two issues are related.

We are using the standard kobo-docker version and we can’t find where to configure django constance:

constance.config.SSRF_ALLOWED_IP_ADDRESS.strip (). split ('\ n')
constance.config.SSRF_DENIED_IP_ADDRESS.strip (). split ('\ n')

We found a commit with the suggested customization on settings.SSRF_OPTIONS as well, however this code is not integrated to the kobo version on master.

At this moment using a ngrok host we can bypass the issue, nevertheless is only a temporaly solution.

To conclude we need referrals about two possible options:
1. where to customize django constance within our kobo-docker version
2. how can we to generate a docker version with the version that already has the suggested customization about allowed_ip_addresses

Thanks for your help.
